### ISSUE TITLE
Error running message queue in 2.6LTS

### ISSUE THREAD

**Issue by LRotherfield** [oroinc/platform#926]
When running `oro:message-queue:consume` in either prod or dev on platform version 2.6 I am getting the following error:

```
Cannot save the cache state date into the database.
SQLSTATE[22007]: Invalid datetime format: 1292 Truncated incorrect DOUBLE value: 'cache'
```

I believe this is happening because the database table `oro_message_queue_state` has an `id` column of type varchar:
```
mysql> show create table oro_message_queue_state\G
*************************** 1. row ***************************
       Table: oro_message_queue_state
Create Table: CREATE TABLE `oro_message_queue_state` (
  `id` varchar(15) COLLATE utf8_unicode_ci NOT NULL,
  `updated_at` datetime DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci
1 row in set (0.00 sec)
```
However the code in the DbalStateDriver is setting the type as `Type:INTEGER` rather than `Type:STRING`: https://github.com/oroinc/platform/blame/2.6/src/Oro/Bundle/MessageQueueBundle/Consumption/StateDriver/DbalStateDriver.php#L109

It looks like this was fixed in platform version 3.1: https://github.com/oroinc/platform/blob/3.1/src/Oro/Bundle/MessageQueueBundle/Consumption/StateDriver/DbalStateDriver.php#L107

I can confirm that changing to `Type:STRING` on line 109 in my local env fixes the issue

Can you confirm this is an issue and then given 2.6 is LTS is this something you guys will patch?

Cheers   
Luke

**Comment by aivus** [comment]
Hi @LRotherfield 

Yes, it looks like a bug in 2.6.

Internal ticket id: BAP-18980

**Comment by mbessolov** [comment]
@LRotherfield  in regards to "given 2.6 is LTS is this something you guys will patch"
please note that, as explained here - https://oroinc.com/b2b-ecommerce/doc/current/community/release-process#orocommerce-community-edition-releases - Oro Team produces patch releases for an LTS version of OroCommerce Community Edition only until the next LTS version is released.

3.1 LTS version has already been released in January, and that means that we are actively publishing community patches only for 3.1 LTS community version.

We may still publish a patch for 2.6 LTS community if it does not require any major work from us.
The best way to facilitate inclusion of the fix in 2.6 would be to create a pull request with a fix **and** accompanying automated tests (or changes to the existing tests).

**Comment by LRotherfield** [comment]
@mbessolov thanks for the clarification, I read the release process page too fast and looked at the "support ends" date for the enterprise version rather than anything to do with the community version.  That is fair enough, I will see if I can produce a fix.

**Comment by LRotherfield** [comment]
@aivus @mbessolov it looks like there were no updates to the message queue bundle with the fix applied in 3.1: https://github.com/oroinc/platform/commit/32cc6246705dfa7c59677f3f3a01d717e9007b78#diff-893218c1efab69b9c2e35302f01c2010
It does not look like there are any additional test cases updated.
So it should be as simple as the above diff, would you still want a pull request?

**Comment by LRotherfield** [comment]
I added a pull request based on my above comment: https://github.com/oroinc/platform/pull/927

**Comment by anyt** [comment]
The issue was fixed in Oro Platform `2.6.40` EE and `3.1.0` versions. 
We are not planning any 2.6 CE releases.

Please consider upgrading.
